libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -65,6 +68,7 @@ public MapboxTelemetry(Context context, String accessToken, String userAgent) {
     initializeTelemetryListeners();
     initializeAttachmentListeners();
     initializeTelemetryLocationState(context.getApplicationContext());
+    this.logger = new Logger();
```

Comment:
why are we creating instance of the class that calls into static method of android `Log`? should we just simply use `Log` directly?